tt_eager/tt_dnn/op_library/groupnorm/groupnorm_op.cpp

```diff
@@ -1496,10 +1511,12 @@ operation::ProgramWithCallbacks groupnorm_sharded_v2_(
         .set_page_size(in_cb_index, in_single_tile_size);
     auto cb_in = tt_metal::CreateCircularBuffer(program, all_cores, in_cb_config);
     // out - stores tilized output
-    uint32_t out_cb_index = CB::c_intermed6;
-    tt_metal::CircularBufferConfig out_cb_config = tt_metal::CircularBufferConfig(in_CB_size, {{out_cb_index, in_data_format}})
-        .set_page_size(out_cb_index, in_single_tile_size);
-    auto cb_out = tt_metal::CreateCircularBuffer(program, all_cores, out_cb_config);
+    if (untilize_out) {
+        uint32_t out_cb_index = CB::c_intermed6;
+        tt_metal::CircularBufferConfig out_cb_config = tt_metal::CircularBufferConfig(in_CB_size, {{out_cb_index, in_data_format}})
+            .set_page_size(out_cb_index, in_single_tile_size);
+        auto cb_out = tt_metal::CreateCircularBuffer(program, all_cores, out_cb_config);
```

Comment:
Learning here. Is`cb_out` just a handle? Buffer's lifetime is controlled by a `program`?

Reply:
yes, they will stay in L1 as long as the kernel is alive